interface/forms/treatment_plan/new.php

```diff
@@ -27,7 +27,6 @@
   "authorized != 0 AND active = 1 ORDER BY lname, fname");
 ?>
 <html><head>
-<?php html_header_show();?>
 
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
 <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker/build/jquery.datetimepicker.min.css">
```

Comment:
@bradymiller Should this be escaped? @morg0n This was already in the code before but now that you are here...